src/core/evaluator.js

```diff
@@ -3550,6 +3550,16 @@ class PartialEvaluator {
             code = unicode;
           }
           break;
+        default:
+          // Support (some) non-standard ligatures.
```

Comment:
Are "non-standard" ligatures equivalent to "stylistic" ones ? (https://en.wikipedia.org/wiki/Ligature_(writing)#Stylistic_ligatures)

Reply:
I don't think so?
We already support normal ligatures such as e.g. `ff`/`ffi`/`ffi`/`ffl` via https://github.com/mozilla/pdf.js/blob/master/src/core/glyphlist.js and https://github.com/mozilla/pdf.js/blob/master/src/core/unicode.js, but the ones added here are "special" ones for which there's no proper unicode-characters.

Basically, this patch tries to better support bad PDF generators.